FreeBSD: building corefx managed components fails on ODBC32 namespace

Dear All,

I am on Windows 10, and I am referring specifically to:
https://github.com/dotnet/corefx/wiki/Building-.NET-Core--2.x-on-FreeBSD#user-content-building-managed-components-on-windows

As well as the Linux counterpart. I tried:
1) `build.cmd` with default flags. This works, but I am well uncertain what to copy to my FreeBSD (11) system.
2) `build-managed.cmd` with the same flags recommended for Linux. The build fails at:

```System.Data.forwards.cs(10,83): error CS0234: The type or namespace name 'ODBC32' does not exist in the namespace 'System.Data.Odbc' (are you missing an assembly reference?)```

3) The same on Ubuntu 16.04, on Hyper-V (using both build.sh and build-managed.sh). Same failure as on Windows. Corefx revision is:
`commit d61b6095568c18cb5e24cff24e0ecf488dfe0f75`

Furthermore, I am uncertain what to do with the binaries, should I be able to produce them. From the following passage, I seem to understand which binaries/libraries are required for testing: should I just move that set of files to `/usr/.../bin` in order to access them globally?

--- a/src/System.Data.Odbc/src/Configurations.props
+++ b/src/System.Data.Odbc/src/Configurations.props
@@ -2,6 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+      netcoreapp2.0-FreeBSD;
       netcoreapp2.0-Linux;
       netcoreapp2.0-OSX;
       netcoreapp2.0-Windows_NT;

--- a/src/System.Data.Odbc/src/System.Data.Odbc.csproj
+++ b/src/System.Data.Odbc/src/System.Data.Odbc.csproj
@@ -113,7 +113,7 @@
       <Link>Common\Interop\Interop.Odbc.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetsLinux)' == 'true'">
+  <ItemGroup Condition="'$(TargetsLinux)' == 'true' OR '$(TargetsFreeBSD)' == 'true'">
     <Compile Include="$(CommonPath)\Interop\Linux\Interop.Libraries.cs">
       <Link>Common\Interop\Linux\Interop.Libraries.cs</Link>
     </Compile>

